pages/13.contributing/01.designer/docs.md

```diff
@@ -14,14 +14,92 @@ taxonomy:
 ---
 
 ---
-Please check out our [Trello team][trello-issues] for tasks to work on!
 
-Cards labeled with Design are relevant to designers - you will find them in the team boards.
+Hello designer, and welcome to the Mautic Community!  At the moment we don't have many designers who are actively contributing to Mautic, so it's a great way to make a big difference.  THANK YOU for considering contributing to Mautic, we're super excited to have you join us!
 
-Many of our Community, Education and Marketing Team tasks will require designer collaboration, please join [#community][community-slack] if you'd like help finding relevant tasks to work on.
+There are several ways you could contribute to Mautic as a designer, and it really depends on what you love doing and how your skills might be best put to use with our current projects.
 
-We also have a public [#design][mautic-design] channel.
+Some areas you may want to consider include:
 
+- Supporting the Product Team by creating visual resources to explain new features
+- Helping the Marketing Team with designing their monthly newsletters
+- Creating new designs for our web resources (for example, the mautic.org website)
+- Working with the Product Team to improve the user experience of Mautic 
+- Contributing to Strategic Initiatives by designing the user experience for new features
+- Improving the user documentation by creating illustrations or images that complement the written information
+- Building a pattern library to standardise the UI patterns used in Mautic
+
+## Getting started
+
+The first step is to join us on our Slack instance.  You can get an invitation [here][slack-invite].    
```

Comment:
I'd add that we use Slack as our main tool for communication, that's why we need new contributors to join our slack channel.

Reply:
I don't know if that would add much info, and if we added it here we should add it on all the other pages. So I'd go for leaving it as it is for the time being as it's the first task, so it would imply being important.

Reply:
Then lets leave it as it is. It was just a small thing that crossed my mind. :)